wiki/Mapping_Techniques/Basics/en.md

```diff
@@ -1,25 +1,104 @@
-Flow of beatmap design
-----------------------
+# Basics
 
-When creating a beatmap, it is a good practice to follow the music's *beat*. As this will help (or harm) players get the idea of what your original intentions were.
+## Common Beatmap Flow
 
-Basic knowledge of mapping
---------------------------
+When starting out with a beatmap, it is good practice to start out by mapping out the music's (or vocal's) beat.
+You will need to determine what will be best for the player as following either can help or harm the beatmap's experience.
 
-Here are the **Basic knowledge of mapping**
+## Symmetry
 
-### Symmetry
+Symmetry is when the beatmap is symmetrical in respect to an axis.
+The most common type of symmetry is horizontal symmetry.
+There are other types of symmetry too, however, like vertical symmetry, diagonal symmetry, and symmetry with respect to the origin.
+Normally we would say, "Think of an odd function", but that would probably alienate a large portion of the playerbase."
 
-Symmetry is when the beatmap is symmetrical in respect to an axis. The most common type of symmetry is horizontal symmetry. There are other types of symmetry too, however, like vertical symmetry, diagonal symmetry, and symmetry with respect to the origin (Normally we would say, "Think of an odd function", but that would probably alienate a large portion of the playerbase).
+![Rotational_Symmetry.png](./Rotational_Symmetry.png "An example of symmetry with respect to the origin")
 
-### Jumps
+To guide you through the process of understanding symmetry, we will go through various examples of symmetry in a map that is already ranked.
+Explaining symmetry in text form (like we are now) isn't as effective without the visual image.
 
-A jump is an advanced mapping technique where the mapper places two hit objects farther apart than the standard distance snap multiplier would allow. This is usually done by either temporarily changing the distance snap multiplier or temporarily turning off distance snap.
+<!-- TODO rewrite the sub-sections for Symmetry -->
```

Comment:
I not remove this, yet, a Brackets extension that I use will list this for me as a thing to do so I don't forget to do this later.